packages/identity/src/identityManager.ts

```diff
@@ -0,0 +1,69 @@
+import { getGlobalAmplitudeNamespace, logger } from '@amplitude/utils';
+import { Identity, IdentityListener, DEFAULT_IDENTITY_INSTANCE } from '@amplitude/types';
+
+import { DefaultIdentity } from './identity';
+
+class IdentityManager {
+  private _instanceMap: Map<string, Identity> = new Map<string, Identity>();
+
+  public getInstance(
+    instanceName: string = DEFAULT_IDENTITY_INSTANCE,
+    optionalFallbackIdentity: Identity | null = null,
```

Comment:
don't know if I want this still - since this is a getter, it's not reliable that you'll set it from here esp since its only a fallback. maybe I only expose using your own identity in `resetInstance`